addons/mail/static/src/core/common/attachment_model.js

```diff
@@ -3,8 +3,9 @@
 import { assignDefined } from "@mail/utils/common/misc";
 
 import { url } from "@web/core/utils/urls";
+import { DiscussModel } from "./discuss_model";
```

Comment:
not liking relative imports at all (unfortunately we lack tooling to ease this I know) but I always manually make them then